pom.xml

```diff
@@ -306,6 +306,17 @@
       <name>Jitpack.io repository</name>
       <!-- needed for brotli-codec -->
     </repository>
+    <repository>
+       <id>staged</id>
+       <name>parquet-staged-releases</name>
+       <url>https://repository.apache.org/content/repositories/staging/</url>
```

Comment:
This is OK to test - we'd want to wait until it's officially released and then remove it before merging

Reply:
+1